tests/test_fields.py

```diff
@@ -408,3 +406,43 @@ def test_null(self):
         for field in fields:
             with self.subTest(field=field):
                 self.assertEqual(getattr(instance, field), None)
+
+
+class TestPGPManager(TestCase):
+    """Test `PGPManager` can be integrated in a `Django` model."""
+    model = EncryptedModelWithManager
+
+    def test_auto_decryption(self):
+        """Assert auto decryption via manager."""
+        expected_string = 'bonjour'
+        expected_date = date(2016, 9, 1)
+        expected_datetime = datetime(2016, 9, 1, 0, 0, 0)
+
+        EncryptedModelFactory.create(
+            digest_field=expected_string,
+            hmac_field=expected_string,
+            pgp_pub_field=expected_string,
+            pgp_sym_field=expected_string,
+            date_pgp_sym_field=expected_date,  # Tests cast sql
+            datetime_pgp_sym_field=expected_datetime,  # Tests cast sql
+        )
+
+        instance = self.model.objects.get()
+
+        # Using `__dict__` bypasses the only the fly decryption for each field
```

Comment:
I think as a non bative English speaker I'm having difficulty to understand this sentence construct `bypasses the only the fly`